ops/ansible/playbook-appserver/bootstrap.yml

```
@@ -0,0 +1,26 @@
+---
+# This playbook modifies the Pi OS image.
+#
+# Usage: this playbook can be run, as follows:
+#
+#    $ ansible-playbook playbook.yml
+
+- hosts: all
+  name: Bootstrap Debian-based OS with Python 3
+  gather_facts: false
+  pre_tasks:
+    - name: Install Python
+      raw: sudo apt-get update -qq && sudo apt-get install -qq python3
+      register: raw_python_install
+      retries: 12
+      delay: 10
+      until: raw_python_install.rc == 0
+
+- hosts: all
+  name: Modify Raspberry Pi OS Image
+  become: true
+  gather_facts: true
+  tasks:
+    - import_role: name=setup_disk_image
+    - import_role: name=base
+    - import_role: name=firmware
```

```suggestion
    - import_role: name=firmware
    - import_role: name=base
```

Logically, this makes more sense, since modifying the firmware is isolated to the `/boot` directory.